airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/spec/DestinationSpecificationInternal.kt

```diff
@@ -18,13 +20,39 @@ import jakarta.inject.Singleton
 class DestinationSpecificationExtender(private val spec: DestinationSpecificationExtension) :
```

Comment:
this diff is a cherry-pick of https://github.com/airbytehq/airbyte/pull/56976, so I can build an image, so I can test the UI on cloud, because local platform dev is broken in multiple ways